x-pack/plugins/reporting/server/types.ts

```diff
@@ -70,7 +67,7 @@ export interface ReportingStartDeps {
   licensing: LicensingPluginStart;
   taskManager: TaskManagerStartContract;
   screenshotting?: ScreenshottingStart;
-  security?: SecurityPluginStart;
+  securityServiceStart: SecurityServiceStart;
```

Comment:
```suggestion
  securityService: SecurityServiceStart;
``` 
or
```suggestion
  securityService?: SecurityServiceStart;
```

Reply:
We might have to use the `security` namespace for the services `start` contract.